packages/tooling/cfp-template/template/src/components/welcome/welcome.ts

```diff
@@ -0,0 +1,26 @@
+import { attr, customElement, FASTElement } from "@microsoft/fast-element";
+import { welcomeStyles as styles } from "./welcome.styles";
+import { welcomeTemplate as template } from "./welcome.template";
+
+export enum Theme {
+    dark = "dark",
+    light = "light",
+}
```

Comment:
We can avoid using enums by [casting as `const` and assigning a type](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums):

```suggestion
export const Theme = {
    dark: "dark",
    light: "light",
} as const;

export type Theme = typeof Theme[keyof typeof Theme];
```

It can also be useful to put static values and types in a separate module - in `fast-foundation` we use a `<component>.options.ts` file. This allows these values to be imported directly, which can help reduce side-effect or execution context collisions (for example, running tests in a `node` context rather than `browser`, which can confuse some tools).

Reply:
I'd prefer an alternative to an enum as well. Personally, I'm not sure I understand yet the value of a static const such as this vs. a static type which also would provide type safety. I'm open, but ideally I'd like to move away from enums as much as we can.

Reply:
> I'm not sure I understand yet the value of a static const such as this vs. a static type which also would provide type safety

A static type can't be used as a value, whereas a static const can.

Reply:
done!